src/vaadin-message.d.ts

```diff
@@ -6,8 +6,9 @@ import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.
  * `<vaadin-message>` is a Web Component for showing a single message with an author, message and time.
  *
  * ```html
- * <vaadin-message foo="bar">
- * </vaadin-message>
+ * <vaadin-message time="2021-01-28 10:43"
+ *     user='{"name":"Bob Ross","abbr":"BR","img":"/test/visual/avatars/avatar.jpg"}'>There is no real ending. It's
```

Comment:
We shouldn't refer to our visual test folder in the documentation.
```suggestion
 *     user='{"name":"Bob Ross","abbr":"BR","img":"/static/img/avatar.jpg"}'>There is no real ending. It's
```